tests/com/twitter/intellij/pants/execution/PantsMakeMessageTest.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.execution;
+
+import junit.framework.TestCase;
+
+import java.util.Optional;
+
+import static com.twitter.intellij.pants.execution.PantsMakeBeforeRun.ERROR_TAG;
+
+public class PantsMakeMessageTest extends TestCase {
+
+  public void testErrorMessageWithLocation() {
+    Optional<PantsMakeBeforeRun.ParseResult> result = PantsMakeBeforeRun.parseErrorLocation(
+      "                       [error] /Users/dummy/workspace/pants_ij/examples/tests/java/org/pantsbuild/example/hello/greet tv/GreetingTest.java:23:1: cannot find symbol",
+      ".java",
+      ERROR_TAG
+    );
+    assertTrue(result.isPresent());
+    assertEquals(
+      "/Users/dummy/workspace/pants_ij/examples/tests/java/org/pantsbuild/example/hello/greet tv/GreetingTest.java",
+      result.get().getFilePath()
+    );
+    assertEquals(23, result.get().getLineNumber());
+    assertEquals(1, result.get().getColumnNumber());
+  }
+
+
+  public void testErrorMessageWithLocation2() {
```

Comment:
Thanks! I will remove having certain file extensions as a requirement, because eventually we will check if it is a valid file path when constructing the hyperlink.